scripts/morphware_data_sizes.py

```diff
@@ -0,0 +1,60 @@
+"""Print the size of the data returned by a Morphware query (data reported)."""
+import json
+import sys
+
+import requests
+
+from telliot_core.queries.morphware import Morphware
+
+
+def main():
+    """
+    Print the size of the data returned by a Morphware query
+    using Morphare-provided data source endpoint:
+
+    curl --request POST http://167.172.239.133:5000/products -H "Content-Type: application/json" \
```

Comment:
Lets update this comment to have the new enpoint
http://167.172.239.133:5000/products-2